### PR TITLE
Milestone #2: Implementing Works and About Me sections of the Mobile version

### PR DESCRIPTION
**Pull request by jodrack07**
During this milestone, I implement the Works and About Me section.
Different things are possible from now like:
- getting the list of the projects
-  getting some details about my experience as a web developer
-  Getting some language stack that I can use to build different applications

Different tools make this implementation possible and working. Below is the list I used to accomplish this milestone:
1. HTML
2. CSS
3. Linters
4. NPM

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -89,6 +89,212 @@ <h1 class="presentation">
...
+          </div>
+          <div class="about-me__description">
+            <p>
+              Hello I’m a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience! If
+              you like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+          </div>
+          <a href="#" class="medias">let's connect</a>
```
Comment (AnselemOdims) [inline index.html]: - [ ] Good job on this but this is supposed to be a header tag `hx`. Kindly change it to be a header tag

File: index.html [hunk 2]
```diff
@@ -89,6 +89,212 @@ <h1 class="presentation">
...
+            </li>
+          </ul>
+          <button type="button" class="btn btn-enabled modalBtn">
+            Get my resume
+          </button>
+        </div>
+
+        <div class="about-me__tools">
+          <div class="tools prog-languages">
+            <h3>languages</h3>
```
Comment (AnselemOdims) [inline index.html]: - [ ] Please kindly ensure the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`

```
<ul>
<li>Languages</li>
<li>Frameworks</li>
<li>Skills</li>
</ul>
```

File: index.html [hunk 3]
```diff
@@ -89,6 +89,212 @@ <h1 class="presentation">
...
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="work__lang">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+            </div>
+            <button type="button" class="btn btn-enabled">See project</button>
```
Comment (AnselemOdims) [inline index.html:117]: - [ ] Kindly make sure the margin between the cards and the sides of the page matches the original design

### Your Design
image

### Original Design
image

File: index.html [hunk 4]
```diff
@@ -89,6 +89,212 @@ <h1 class="presentation">
...
+        <div class="about-me-container">
+          <div class="about-me__title">
+            <h2>About Myself</h2>
+          </div>
+          <div class="about-me__description">
+            <p>
+              Hello I’m a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience! If
+              you like what you see and have a project you need coded, don’t
+              hestiate to contact me.
```
Comment (AnselemOdims) [inline index.html:212]: - [ ] Please ensure that the padding at the top of this section has enough room. At the moment, it does not exactly match the original design.

### Your Design
image

### Original Design
image

File: index.html [hunk 5]
```diff
@@ -89,6 +89,212 @@ <h1 class="presentation">
...
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  src="images/icons/github.svg"
+                  alt="github"
+                  title="follow me on github"
+              /></a>
+            </li>
```
Comment (AnselemOdims) [inline index.html]: - [ ] Please ensure that this section matched the original design guidelines (margins, paddings, spacing etc). At the moment, the original design is centered while yours is not
- [ ] Also try and prevent the horizontal scrolling as this does not create a good user experience.

### Your Design
image

### Original Design
image

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @joseph07-drack , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by karvel-code — APPROVED** [review]
Hi @joseph07-drack ,

## Status: APPROVED!!! 🥇 💯  
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**